### PR TITLE
Better support for default branding styles

### PR DESCRIPTION
**Pull request by xjensen**
Per #115, I've changed several default values in ds-branding. These changes are necessary to implement a mostly default experience on drought.ca.gov. There are quite a few changes to font settings and margin/padding. See changelog (and diffs!) for details. Happy to discuss further as needed.

I will shortly provide another PR for headless.cannabis.ca.gov to override these new defaults on that site, to try preventing regressions over there.

### CODE REVIEW

File: components/branding/src/index.scss [hunk 1]
```diff
@@ -1,15 +1,9 @@
-/* Cannabis logo font */
```
Comment (aaronhans) [inline components/branding/src/index.scss:1]: nice yeah this is cannabis specific so should only be on cannabis

**Review by aaronhans — APPROVED** [review]
looks good, thanks for getting this to conform with drought designs too

**Comment by xjensen** [comment]
I've come up with a solution that works across both Cannabis and Drought, with no overrides, using CSS Grid. It's a significant rewrite of many parts of the component, but *mostly* looks the same. I have more testing and documentation to do here. More to come.

**Review by kkoryaka — APPROVED** [review]

**Comment by kkoryaka** [comment]
I Looks good. Will you do search box button color update in the next revision?

**Review by aaronhans — APPROVED** [review]
Love the css grid usage, really cool that we can now take advantage of stuff like that now.

The HTML duplication with logo/no logo versions is unfortunate but I don't have any better ideas.

Congratulations on removing the overrides from the implementing site, that is a big win!

**Comment by xjensen** [comment]
> I Looks good. Will you do search box button color update in the next revision?

Thanks for reminder, @kkoryaka! I forgot to move this over from my test code. I've added a new commit to address this.

**Pull request merged**